src/components/va-on-this-page/va-on-this-page.tsx

```diff
@@ -16,18 +16,18 @@ export class VaOnThisPage {
     >;
 
     return (
-      <nav>
-        <h2>On this page</h2>
-        <ul>
+      <nav aria-label="table of contents">
+        <dl>
+        <dt id="on-this-page">On this page</dt>
+        <dd role="definition">
           {h2s.map(heading => (
-            <li>
-              <a href={`#${heading.id}`}>
-                <i aria-hidden="true" class="fas fa-arrow-down"></i>
-                {heading.innerText}
-              </a>
-            </li>
+            <a href={`#${heading.id}`}>
+              <i aria-hidden="true" class="fas fa-arrow-down"></i>
+              {heading.innerText}
+            </a>
           ))}
-        </ul>
+        </dd>
+        </dl>
```

Comment:
Mmm, GitHub won't let me add a suggestion here, but can you indent the children of the `<dl>`? This should have been caught / fixed by linting, but :man_shrugging:. I'll look into that later.